src/github-handler/comment-handler/get-hunk-scope-handler/remote-patch-ranges-handler.ts

```diff
@@ -78,62 +78,50 @@ index cac8fbc..87f387c 100644
-    try {
-      const validHunks = parseHunks(_DIFF_HEADER + patch);
-      // TODO: consider returning hunks instead
-      const validLineRanges = validHunks.map(hunk => {
-        return {start: hunk.newStart, end: hunk.newEnd};
-      });
-      allValidLineRanges.set(filename, validLineRanges);
-    } catch (err) {
-      logger.info(
-        `Failed to parse the patch of file ${filename}. Resuming parsing patches...`
-      );
-    }
-  });
-  return allValidLineRanges;
-}
-
-/**
- * For a pull request, get each remote file's current patch range to identify the scope of each patch as a Map,
- * as well as a list of files that cannot have suggestions applied to it within the Pull Request.
- * The list of files are a subset of the total out-of-scope files.
+ * For a pull request, get each remote file's current patch range to identify the scope of each patch as a Map.
  * @param {Octokit} octokit the authenticated octokit instance
  * @param {RepoDomain} remote the remote repository domain information
  * @param {number} pullNumber the pull request number
  * @param {number} pageSize the number of files to return per pull request list files query
- * @returns {Promise<Object<Map<string, Range[]>, string[]>>} the scope of each file in the pull request and the list of files whose patch data could not be resolved
+ * @returns {Promise<Map<string, Hunk[]>>} the scope of each file in the pull request
  */
-export async function getPullRequestScope(
+export async function getPullRequestHunks(
   octokit: Octokit,
   remote: RepoDomain,
   pullNumber: number,
   pageSize: number
-): Promise<{validFileLines: Map<string, Range[]>; invalidFiles: string[]}> {
-  try {
-    const {patches, filesMissingPatch} = await getCurrentPullRequestPatches(
-      octokit,
-      remote,
-      pullNumber,
-      pageSize
-    );
-    const validFileLines = patchTextToRanges(patches);
-    return {validFileLines, invalidFiles: filesMissingPatch};
-  } catch (err) {
+): Promise<Map<string, Hunk[]>> {
+  const files = (
+    await octokit.pulls.listFiles({
+      owner: remote.owner,
+      repo: remote.repo,
+      pull_number: pullNumber,
+      per_page: pageSize,
+    })
+  ).data;
+  const pullRequestHunks: Map<string, Hunk[]> = new Map();
+  if (files.length === 0) {
     logger.error(
-      'Could not convert the remote pull request file patch text to ranges'
+      `0 file results have returned from list files query for Pull Request #${pullNumber}. Cannot make suggestions on an empty Pull Request`
+    );
+    throw Error('Empty Pull Request');
+  }
+  files.forEach(file => {
+    if (file.patch === undefined) {
+      // files whose patch is too large do not return the patch text by default
+      // TODO handle file patches that are too large
+      logger.warn(
+        `File ${file.filename} may have a patch that is too large to display patch object.`
+      );
+    } else {
+      const hunks = parseHunks(_DIFF_HEADER + file.patch);
```

Comment:
it seems like this new logic is biting you, with regards to windows line endings.

In test you might want to add a helper that just standardizes on `\n`.